app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php

```diff
@@ -442,115 +455,6 @@ protected function _getCustomerGroupCode($customer)
         }
     }
 
-   /* ########### THE CODE BELOW IS NOT USED ############# */
```

Comment:
lets keep this comment or remove everything below it.

Reply:
Old code is deleted, isn't it?

Reply:
Yes, it is :)

Reply:
not all. " public function parse()" is not deleted

Reply:
@tmotyl this was a code block/mehtod "OLD_something()" ... this has been deleted. All other dead code is untouched and will be remoded later. Please accept for now.

Reply:
ok